app/models/forms/risk/risk_from_others.rb

```diff
@@ -1,22 +1,27 @@
 module Forms
   module Risk
     class RiskFromOthers < Forms::Base
-      CSRA_TOGGLE_CHOICES = %w[ high standard unknown ]
+      CSRA_HIGH = 'high'
+      CSRA_STANDARD = 'standard'
+      CSRA_TOGGLE_CHOICES = [CSRA_HIGH, CSRA_STANDARD, DEFAULT_CHOICE]
+
+      def self.configure_csra_section
```

Comment:
That commit was just an attempt at grouping(showing they have a relationship) those two things which feel kind of inextricably linked in the form and as the CSRA section is slightly different it can't be configured using the macro. Alternative way of achieving the same thing could be to use something like ``` concerning :ConfigureCsraSection do ..` or left alone.

This addition was partly influenced by missing the details of this particular section of the form when I was building the reset attribute stuff the other day.